lerobot/common/policies/normalize.py

```diff
@@ -78,10 +79,14 @@ def create_stats_buffers(
             # https://github.com/huggingface/safetensors/blob/079781fd0dc455ba0fe851e2b4507c33d0c0d407/bindings/python/py_src/safetensors/torch.py#L97.
             if mode == "mean_std":
                 buffer["mean"].data = stats[key]["mean"].clone()
-                buffer["std"].data = stats[key]["std"].clone()
+                buffer["std"].data = stats[key]["std"].clone().clamp_min(std_epsilon)
```

Comment:
Hmm, so I think that if the buffer says "std" we should trust that it is that, and not a modified version of it. Same with "max" below. I can think of an example where this would have been important given an experience of my own: when porting weights from the original work, I take their stats and set them in the state dict here. I would really hope that their stats are true to their names, and I would assume ours are.

If you agree, my ask would be to move the clamping logic to the normalization function.

Reply:
Fine with me. Did the clamping when storing instead of computing for a small speed-up improvement but it's probably an epsilon speedup anyway (pun intended)

Reply:
updated following this suggestion. feel free to check and merge if ok